idpi/test/test_destagger.py

```diff
@@ -0,0 +1,71 @@
+import os
+import shutil
+import subprocess
+
+import grib_decoder
+import jinja2
+import numpy as np
+import xarray as xr
+from operators.destagger import destagger
+
+
+def test_destagger():
+    datadir = "/project/s83c/rz+/icon_data_processing_incubator/data"
+    datafile = datadir + "/lfff00000000"
+    cdatafile = datadir + "/lfff00000000c"
+
+    ds = {}
+    grib_decoder.load_data(ds, ["U", "V"], datafile, chunk_size=None)
+    grib_decoder.load_data(ds, ["HHL"], cdatafile, chunk_size=None)
+
+    U = destagger(ds["U"], "x")
+    V = destagger(ds["V"], "y")
+    HFL = destagger(ds["HHL"], "generalVerticalLayer")
+
+    conf_files = {
+        "inputi": "/scratch/cosuna/fieldextra_tests/lfff<DDHH>0000",
+        "inputc": "/scratch/cosuna/fieldextra_tests/lfff00000000c",
+        "output": "<HH>_destagger.nc",
```

Comment:
I suppose that we should replace the paths to /scratch/cosuna.

Reply:
yes, I forgot these paths when pushing the first commit. I fixed already in the PR to reduce the domain sizes of data: 
https://github.com/MeteoSwiss-APN/icon_data_processing_incubator/pull/26/files

Reply:
OK!